site/content/deployment/enmasse.md

```diff
@@ -0,0 +1,56 @@
++++
+title = "OpenShift"
+weight = 475
++++
+
+## Build Hono
+
+~~~sh
+~/hono$ mvn clean install -Pbuild-docker-image
+~~~
+
+## Deploy EnMasse
+
+Dowload the release, unpack and run
+
+~~~sh
+./deploy-openshift.sh -n hono
```

Comment:
where does this script come from?

Reply:
It's the deployment script provided by EnMasse for OpenShift. It's inside the tar file you download from the releases page. @dejanb if you are using minishift, should you specify the -m parameter with https://[minishiftip]:8443 ?

Reply:
From the enmasse release distro (https://github.com/EnMasseProject/enmasse/releases/). I didn't describe things in details yet, but this should be a default way of deploying enmasse.

Reply:
@ppatierno I'll add it

Reply:
Is it planned to have this integration deployed also to Kubernetes?

Reply:
@balsmn yes, we can do that as well. I think it shouldn't be too much different then this